src/test/ui/const-generics/raw-ptr-const-param.rs

```diff
@@ -1,3 +1,6 @@
+// normalize-stderr-64bit "0x00000000" -> "0x[PREFIX]"
```

Comment:
Can't you normalize `0x0{8}([0-9a-f]{8})` to `0x\1`, instead?
Or does it not expose regex replacements like that?

Reply:
I kinda had this before, but thought it odd to make the prefix disappear on 64 bit without leaving any kind of note

Reply:
I mean, it's `0`. We might not even want to print the top 8 zeros in the compiler itself.